### PR TITLE
Feature 54153 translations

### PR DESCRIPTION
**Pull request by TheWalkingLeek**

### CODE REVIEW

File: frontend/app/helpers/t.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+export default Helper.extend({
+  intl: service(),
+
+  compute(params) {
+    return this.intl.t(`${this.translationKeyPrefix()}.${params[0]}`);
+  },
+
+  translationKeyPrefix() {
+    return this.intl.locale[0].split("-")[0];
+  }
```
Comment (lucashabersaat) [inline frontend/app/helpers/t.js:13]: Nice :fireworks:

File: frontend/translations/de.yaml [hunk 2]
```diff
@@ -0,0 +1,456 @@
...
+    weak: Das Passwort ist unsicher
+    good: Das Passwort genügt
+    strong: Das Passwort ist stark!
+
+  #Pundit
+  pundit:
+    default: Zugriff verweigert
+    team_policy:
+      update?:  Du bist nicht Mitglied dieses Teams
+      destroy?: Nur Admins dürfen Teams löschen
```
[the remark this answers is not in the document]
Comment (TheWalkingLeek) [reply frontend/translations/de.yaml:456]: It's not missing, on line 344

File: frontend/translations/fr.yml [hunk 3]
```diff
@@ -0,0 +1,455 @@
...
+    weak: Le mot de passe est faible
+    good: Le mot de passe est bon
+    strong: Le mot de passe est forte!
+
+  #Pundit
+  pundit:
+    default: Accès refusé
+    team_policy:
+      update?: Vous n'êtes pas membre de cette équipe
+      destroy?: Seul l'administrateur peut supprimer des équipes
```
[the remark this answers is not in the document]
Comment (TheWalkingLeek) [reply frontend/translations/fr.yml:455]: Same here, on line 342

File: frontend/app/helpers/t.js (hunk 1 above)
Comment (psunix) [reply frontend/app/helpers/t.js:13]: :dancers:

File: frontend/app/routes/application.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import Route from "@ember/routing/route";
+import { inject as service } from "@ember/service";
+
+export default class IndexRoute extends Route {
+  @service intl;
+
+  beforeModel() {
+    /* eslint-disable no-undef */
+    let selectedLocale =
+      $(".container form option[selected='selected']")[0].value || "en";
```
Comment (psunix) [inline frontend/app/routes/application.js:10]: nice !
the DOM is ours :smiling_imp:

**Pull request merged**